content/blog/how-to-backup-and-restore-stateful-applications-on-kubernetes-using-kasten-k10-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,660 @@
+service/primera3par-csp-svc   ClusterIP   10.101.42.76    <none>        8080/TCP   56d
+
+NAME                          DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR   AGE
+daemonset.apps/hpe-csi-node   1         1         1       1            1           <none>          56d
+
+NAME                                  READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/hpe-csi-controller    1/1     1            1           56d
+deployment.apps/nimble-csp            1/1     1            1           56d
+deployment.apps/primera3par-csp       1/1     1            1           56d
+deployment.apps/snapshot-controller   2/2     2            2           56d
+
+NAME                                             DESIRED   CURRENT   READY   AGE
+replicaset.apps/hpe-csi-controller-54cf448d85    1         1         1       56d
+replicaset.apps/nimble-csp-74d57f9487            1         1         1       56d
+replicaset.apps/primera3par-csp-59f5dfc499       1         1         1       56d
+replicaset.apps/snapshot-controller-5fd799f6b5   2         2         2       56d
+```
+
+HPE CSI driver for K8s supports both dynamical persistent volumes and volume snapshots. The following are the *StorageClasses* and the *VolumeSnapshotClass* that are configured in the cluster: 
+
+```shell
+$ kubectl get storageclasses
+NAME                                 PROVISIONER                    RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
+gl-sbc-hpe                           csi.hpe.com                    Delete          Immediate              true                   56d
+gl-sbp-frank-gl1-sstor01 (default)   csi.hpe.com                    Delete          Immediate              true                   56d
+hpe-hdd-storage                      kubernetes.io/no-provisioner   Delete          WaitForFirstConsumer   false                  56d
+hpe-nvme-storage                     kubernetes.io/no-provisioner   Delete          WaitForFirstConsumer   false                  56d
+hpe-ssd-storage                      kubernetes.io/no-provisioner   Delete          WaitForFirstConsumer   false                  56d
+
+$ kubectl  get volumesnapshotclasses
+NAME                                 DRIVER        DELETIONPOLICY   AGE
+gl-sbp-frank-gl1-sstor01             csi.hpe.com   Delete           56d
+```
+
+[The joint partnership between HPE and Veeam](https://www.kasten.io/kubernetes/resources/blog/kubernetes-backup-with-hpe-csi-and-kasten-k10) supports HPE CSI driver for K8s and Kasten K10 as a data management solution for K8s backup and recovery. The following sections will show you how to install Kasten K10 on the cluster and how to use it with HPE CSI driver for K8s to backup and restore the persistent volumes of the stateful applications running in the cluster using volume snapshots. 
+
+### Prerequisites
+
+Before starting, make sure you meet the following requirements:
```

Comment:
Similar to the other blog post, I'd recommend rephrasing this to:

"Before starting, make sure you have the following:"